addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_ubl_be.py

```diff
@@ -242,7 +242,7 @@ def test_import_export_invoice_xml(self):
 
         # Import:
         created_bill = self.env['account.move'].create({'move_type': 'in_invoice'})
-        created_bill.message_post(attachment_ids=[attachment.id])
+        created_bill.with_context(default_move_type=created_bill.move_type).message_post(attachment_ids=[attachment.id])
```

Comment:
Why do you need to put the journal in the context if the journal is already set?